src/explore-education-statistics-admin/src/pages/bau/BoundaryDataPage.tsx

```diff
@@ -0,0 +1,198 @@
+import Page from '@admin/components/Page';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import React from 'react';
+import FormattedDate from '@common/components/FormattedDate';
+import Details from '@common/components/Details';
+import Link from '@admin/components/Link';
+import { useQuery } from '@tanstack/react-query';
+import boundaryDataQueries from '@admin/queries/boundaryDataQueries';
+import classNames from 'classnames';
+import styles from './BoundaryDataPage.module.scss';
+
+const BoundaryDataPage = () => {
+  const { data: boundaryLevels = [], isLoading: isLoadingBoundaryLevels } =
+    useQuery(boundaryDataQueries.getBoundaryLevels);
+
+  return (
+    <Page
+      title="Boundary data"
+      caption="Manage Boundary data"
+      wide
+      breadcrumbs={[
+        { name: 'Platform administration', link: '/administration' },
+        { name: 'Boundary data' },
+      ]}
+    >
+      <LoadingSpinner
+        loading={isLoadingBoundaryLevels}
+        text="Loading boundary levels"
+      >
+        <Details summary="Glossary" id="glossary">
+          <h2 className="govuk-heading-lg">Boundary types</h2>
+          <dl
+            className={classNames(styles.glossarySection, styles.boundaryTypes)}
+          >
+            <dt>BFC</dt>
```

Comment:
Is there any chance the frontend is going to want access to these boundary type descriptions in other places? If so I'd consider creating something like `locationsLevelsMap` for boundary types, and then using that to display them here.